src/frontend/src/pages/Chapter/Chapter.controller.tsx

```diff
@@ -69,10 +70,14 @@ export const Chapter = () => {
   })
   if (counter >= 20) badgeUnlocked = true
 
-
   useEffect(() => {
-    if (user) dispatch(getUser({ username: user.username }))
+    if (user && !isUserFetched) {
+      dispatch(getUser({ username: user.username }))
+      setIsUserFetched(true)
+    }
+  }, [dispatch, user, isUserFetched])
```

Comment:
@petarvujovic98  Do you understand what's happening here? I don't. So it probably needs better variable names or comments. What is `getUser` doing? It seems weird that we'd only want to call `getUser` if `user` already has a value.

Reply:
@ryancwalsh It seems to me that `getUser` is fetching the user record from the backend via the `getPublicUser` function. But the data it retrieves then is the same `user` data that instantiated the action. The only reason I think this to be the case is to check for authentication validity, or more specificaly implicitly check if the JWT is valid through refetching the user record.

Reply:
@petarvujovic98 I still don't understand. And the reasons I'm pressing are:

1. I don't want to approve something and deploy to production if we don't know that this is the right fix, especially since there is no known bug right now (and it's just an eslint warning)
2. I want the code to either be more self-documenting or have helpful comments

But (as you know) this issue is super low priority, so you can continue focusing on WT or whatever for now.

Reply:
What is the Merkle Leaf